src/browser-polyfill.js

```diff
@@ -382,36 +396,96 @@ if (typeof browser === "undefined") {
           return false;
         }
 
+        // A small helper to send the message if the promise resolves
+        // and an error if the promise rejects (a wrapped sendMessage has
+        // to translate the message into a resolved promise or a rejected
+        // promise).
+        const sendPromisedResult = (promise) => {
+          promise.then(msg => {
+            // send the message value.
+            sendResponse(msg);
+          }, error => {
+            console.error(error);
+            // Send a JSON representation of the error if the rejected value
+            // is an instance of error, or the object itself otherwise.
```

Comment:
This last bit of the comment "or the object itself otherwise" no longer applies?